scripts/remote_customize_entrypoint.sh

```diff
@@ -109,7 +90,8 @@ fi
 
 # unpack the image file
 mkdir -p $IMAGE_ROOT_PARENT
-unsquashfs -f -d $IMAGE_ROOT_DIR /data/image.sqsh
+unsquashfs -f -d $IMAGE_ROOT_DIR ${IMAGE_ROOT_PARENT}/image.sqsh
```

Comment:
This is not related to your current PR, so feel free to dismiss just on that basis. I just noticed that this script (and the others too, I think) are not running with `set -e`, which means that command failures won't automatically end script execution. This is not uncommon, but then I noticed here, for example, that the unsquashfs return code is not checked. I do see other cases in this and other scripts where return codes are checked, so I just wanted to make sure that the omission here was deliberate.